packages/react-components/react-slider/library/src/components/Slider/__snapshots__/Slider.test.tsx.snap

```diff
@@ -4,7 +4,7 @@ exports[`Slider renders disabled Slider correctly 1`] = `
 <div>
   <div
     class="fui-Slider"
-    style="--fui-Slider--direction: 90deg; --fui-Slider--progress: 50%;"
+    style="--fui-Slider--direction: 90deg; --fui-Slider--progress: 50%; --fui-Slider--progress-value: 0.5;"
```

Comment:
can we add tests (probably vr test?) that cover this new implementation ?

Reply:
the PR says feat, the change file says minor = it's a new feature. 

I'm not really sure how adding new css variable to snapshots guarantees that our new stuff works as expected ?

Reply:
> I'm not really sure how adding new css variable to snapshots guarantees that our new stuff works as expected ?

This particular change does not ensure any new functionality, as it merely updates an existing snapshot. However, the current VR tests for the component should confirm that the new features, such as the slider's thumb and position changes, are functioning as intended. Additionally, I have reviewed the VR tool report for accuracy.

Reply:
Updated PR, this is no longer relevant